index.html

```diff
@@ -1497,37 +1624,60 @@
             presentation connection</dfn> using <em>connection</em>, it MUST do
             the following:
           </p>
+          <dl>
+            <dt>
+              Input
+            </dt>
+            <dd>
+              <code>presentationConnection</code>, the <a>presentation
+              connection</a> to be closed
+            </dd>
+            <dd>
+              <code>closeReason</code>, the
+              <code>PresenentationConnectionClosedReason</code> describing why
```

Comment:
Typo: "PresenentationConnectionClosedReason" should be "PresentationConnectionClosedReason"